modules/programs/mbsync.nix

```diff
@@ -70,19 +73,22 @@ let
           Inbox = "${maildir.absPath}/${folders.inbox}";
           SubFolders = "Verbatim";
         }
-        //
-        optionalAttrs (mbsync.flatten != null) { Flatten = mbsync.flatten; }
+        // mbsync.extraConfig.store
```

Comment:
Best have this last, i.e., after the `// optionalAttrs (mbsync.flatten != null) { Flatten = mbsync.flatten; }` line.